discord/ui/view.py

```diff
@@ -381,7 +381,7 @@ async def on_timeout(self) -> None:
         """
         pass
 
-    async def on_error(self, error: Exception, item: Item[Any], interaction: Interaction) -> None:
+    async def on_error(self, error: Exception, interaction: Interaction, item: Item[Any]) -> None:
```

Comment:
```suggestion
    async def on_error(self, interaction: Interaction, error: Exception, item: Item[Any]) -> None:
```

Interaction should always be first.

Reply:
Should `error` be between `interaction` and `item`, or be last?

Reply:
Amusingly, `Tree.on_error` does `(interaction, command, error)`.

I am not a fan of three parameter callbacks. For that one in particular it might be worth removing `command` altogether since there's `Interaction.command` now.

Reply:
This is, I think, beyond the scope of this PR, as here Interaction was clearly not before Item. I've applied the suggested changes.

Reply:
Yeah, it is outside of the scope of the PR. It's just a relevant musing.